c/frame/py_frame.cc

```diff
@@ -160,6 +188,26 @@ void Frame::_clear_types() const {
 }
 
 
+oobj Frame::head(const PKArgs& args) {
+  size_t n = args[0].is_undefined()? 10 : args[0].to_size_t();
+  if (n > dt->nrows) n = dt->nrows;
+  py::otuple aa(2);
+  aa.set(0, py::orange(0, n, 1));
+  aa.set(1, py::None());
```

Comment:
What this `py::None()` corresponds to?

Reply:
that's `None` in python. So, I'm creating a tuple `(range(0, n), None)` here.